src/validates/reset-password.schema.js

```diff
@@ -0,0 +1,26 @@
+import * as Yup from 'yup'
+
+import { EMAIL_REGEX, PHONE_REGEX } from '@/constants'
+
+// constants
+const MAX_LEN_USERNAME = 50
+
+const FORM_RESET_PASSWORD = {
+  VERIFICATION: 'verification',
```

Comment:
hình như em bị nhầm rồi
cái verification là người dùng chỉ nhấn vào đường link và nó tự điền thôi
còn form của mình phải bao gồm:
OLD_PASSWORD
NEW_PASSWORD
CONFIRM_PASSOWORD
=> tất nhiên là form thì yêu cầu nhiều vậy, nhưng khi request BE thì chỉ cần gửi 2 cái thôi, confirmPassword là để check ở phía FE